_sass/_base.scss

```diff
@@ -11,6 +11,27 @@ dl, dd, ol, ul, figure {
 
 
 /**
+ * Variables
+ */
+$navy-blue-color: rgb(22,33,63);
```

Comment:
Each selector in a comma sequence should be on its own line<br>Commas in function arguments should be followed by a single space